react-web/src/components/Header/Header.tsx

```diff
@@ -38,7 +38,7 @@ const Header = () => {
 
   useEffect(() => {
     if (isLoggedIn) {
-      navigate("/");
```

Comment:
is this effect doing anything now?

Reply:
Doesn't seem like it, but I want to keep the useEffect block commented for reference, just until all internal navigation bugs are resolved